lib/builderator/config/defaults.rb

```diff
@@ -95,33 +98,75 @@ module Config
         end
       end
 
-      generator.gemfile.vagrant do |vagrant|
-        vagrant.install true
-        vagrant.version 'v1.8.0'
-      end
+      generator.project :default do |base|
+        base.ruby.version '2.1.5'
```

Comment:
Is there a reason for this?  Are we only running the test suite against this version?  I've seen developers struggle with this constraint.